### ISSUE TITLE
Use different start positions

### ISSUE THREAD

**Issue by krichter722** [amitguptagwl/fast-lorem-ipsum#3]
It'd be nice to get characters, words and sentences starting from different parts of base text data.

**Comment by amitguptagwl** [comment]
I'm not sure if there is any real use case of that but this should be the
easy change to do that.

**Comment by jakeparis** [comment]
Yes, there's a real use case for this. I'd like to create 4 blocks of dummy text, and it would look better if they were all distinct.
